examples/plot_function_counter.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""
+======================================
+Identifying function names in a script
+======================================
+
+Calls Sphinx-Gallery identify names function to figure out which
+functions are called in the script and to which module do they belong.
+"""
+
+# Code source: Óscar Nájera
+# License: BSD 3 clause
+
+import matplotlib.pyplot as plt
+import sphinxgallery.backreferences as spback
+
+
+filename = spback.__file__.replace('.pyc', '.py')
+file_code = open(filename, 'r').read()
```

Comment:
Well I wanted something with long enough to produce a figure, and code that actually got executed. That's why the first idea was to read the file itself.